tests/conftest.py

```diff
@@ -341,20 +341,31 @@
 ]
 
 
+def only_windows(fn: Callable) -> Callable:
+    """
+    Decorate a pytest method to run a test only in a case we are on Windows.
+
+    :return: decorated method.
+    """
+    return action_for_platform("Windows", skip=False)(fn)
+
+
 def skip_test_windows(fn: Callable) -> Callable:
     """
     Decorate a pytest method to skip a test in a case we are on Windows.
 
     :return: decorated method.
     """
-    return skip_for_platform("Windows")(fn)
+    return action_for_platform("Windows", skip=True)(fn)
 
 
-def skip_for_platform(platform_name: str) -> Callable:
+def action_for_platform(platform_name: str, skip: bool = True) -> Callable:
```

Comment:
Not using skip?

Reply:
I needed it because some tests migth be run ONLY on specific platforms. I used the `skip` argument to switch between the two behaviours

Reply:
yes, not yet, will fix that